src/HydraAuction/Runner.hs

```diff
@@ -203,27 +207,29 @@ executeTestRunner runner = do
   withTempDir "test-hydra-auction" $ \tmpDir -> do
     let stateDirectory = MkStateDirectory tmpDir
     tracerForCardanoNode <- fileTracer stateDirectory
-    let tracer = contramap show stdoutTracer
     withCardanoNodeDevnet
       (contramap (FromHydra . FromCardanoNode) tracerForCardanoNode)
       tmpDir
       $ \node ->
-        executeRunner
-          (MkExecutionContext {tracer = tracer, node = node, actor = Alice})
-          runner
+        executeRunnerWithNodeAs node Alice runner
 
-localNode :: RunningNode
-localNode =
+dockerNode :: RunningNode
+dockerNode =
   RunningNode
     { networkId = Testnet $ NetworkMagic 42
     , nodeSocket = "./devnet/node.socket"
     }
 
-executeRunnerWithLocalNode :: forall x. Runner x -> IO x
-executeRunnerWithLocalNode runner = do
+executeDockerRunner :: Runner () -> IO ()
```

Comment:
That it very strange function. `docker-compose` starts the whole cluster, not only L1 node.